pac/README.md.revA.template

```diff
@@ -0,0 +1,9 @@
+# Peripheral Access Crate ${crate} for ${mcu} 32-bit MCU from Atmel/Microchip
+
+This PAC crate is related with **Hardware Revision A** that's highely likely outdated and no loger in production.
```

Comment:
```suggestion
This PAC crate relates to **Hardware Revision A** that has been superseded by **Hardware Revision B** (refer to the `${crate}b` crate). The former is no longer in production.
```

Expansion untested. A link to the B revision crate would be more apt.

Reply:
Change is coming with the link to `crates.io`. I'm not pushing regenerated PACs as I already have newer `svd2rust` and it's >33k files - so we do that once we do new PACs release.